src/proto/ctl/smd.proto

```diff
@@ -149,19 +161,17 @@ message SmdQueryReq {
 	string target = 7; // response should only include information about this VOS target
 	bool no_reint = 9; // specify if device reint is needed (used for replace cmd)
 	bool identify = 10; // set the VMD LED state to quickly blink
-	uint32 state_mask = 11; // show only devices with state matching bit-mask
 	string replace_uuid = 12; // UUID of new device to replace storage with
+	bool reset_led = 13; // for resetting VMD LED, debug only
+	bool get_led = 14; // get LED state of VMD devices
+	string tr_addr = 15; // constrain query to this PCI address (device only)
```

Comment:
transport address is what is used throughout SPDK, PCI address is one type of transport address, others being NVMoF. I'm using the transport just because sometime in the future we are supposed to be supporting NVMoF

Reply:
In that case, it might be better to update the comment.

Reply:
done